typescript/src/actions/evm/types.ts

```diff
@@ -1,48 +1,76 @@
 import type {
   WaitForUserOperationOptions,
   WaitForUserOperationReturnType,
 } from "./waitForUserOperation.js";
 
-type Actions<T extends EvmAccount | EvmSmartAccount> = {
+type Actions = {
   /**
-   * Transfer an amount of a token from an account to another account.
+   * List the token balances of an account.
    *
-   * @param options - The options for the transfer.
-   * @param options.to - The account or 0x-prefixed address to transfer the token to.
-   * @param options.amount - The amount of the token to transfer.
-   * @param options.token - The token to transfer.
-   * @param options.network - The network to transfer the token on.
+   * @param options - The options for the list token balances.
+   * @param options.network - The network to list the token balances on.
    *
-   * @returns The result of the transfer.
+   * @returns The result of the list token balances.
    *
    * @example
    * ```typescript
-   * const { status } = await sender.transfer({
-   *   to: "0x9F663335Cd6Ad02a37B633602E98866CF944124d",
-   *   amount: "0.01",
-   *   token: "usdc",
+   * const balances = await account.listTokenBalances({
    *   network: "base-sepolia",
    * });
    * ```
+   */
+  listTokenBalances: (
+    options: Omit<ListTokenBalancesOptions, "address">,
+  ) => Promise<ListTokenBalancesResult>;
+
+  /**
+   * Requests funds from an EVM faucet.
+   *
+   * @param {RequestFaucetOptions} options - Parameters for requesting funds from the EVM faucet.
+   * @param {string} options.network - The network to request funds from.
+   * @param {string} options.token - The token to request funds for.
+   * @param {string} [options.idempotencyKey] - An idempotency key.
+   *
+   * @returns A promise that resolves to the transaction hash.
+   *
+   * @example
+   * ```ts
+   * const result = await account.requestFaucet({
+   *   network: "base-sepolia",
+   *   token: "eth",
+   * });
+   * ```
+   */
+  requestFaucet: (options: Omit<RequestFaucetOptions, "address">) => Promise<RequestFaucetResult>;
+};
+
+export type AccountActions = Actions & {
+  /**
+   * Transfer an amount of a token from an account to another account.
+   *
+   * @param options - The options for the transfer.
+   * @param options.to - The account or 0x-prefixed address to transfer the token to.
+   * @param options.amount - The amount of the token to transfer represented as an atomic unit (see examples).
+   * @param options.token - The token to transfer.
+   * @param options.network - The network to transfer the token on.
+   *
+   * @returns An object containing the transaction hash.
    *
    * @example
-   * **Pass a bigint value**
    * ```typescript
-   * const { status } = await sender.transfer({
+   * import { parseUnits } from "viem";
+   *
+   * const { transactionHash } = await sender.transfer({
    *   to: "0x9F663335Cd6Ad02a37B633602E98866CF944124d",
-   *   amount: 10000n, // equivalent to 0.01 usdc
+   *   amount: parseUnits("0.01", 6),
```

Comment:
I think this is less clear than just specifying atomic units directly

Reply:
👍 Updated these docstrings with more info